### ISSUE TITLE
v0.7.0 fails to compile with `transmute called with types of different sizes` on armv7-linux-androideabi

### ISSUE THREAD

**Issue by passy** [prevoty/jni-rs#49]
Hi! Apologies if I'm missing something obvious.

It seems that as of b6463aac942ecb25a99cb30e57a481ee7e1852a3 I can no longer compile the library for arm targets.

Trying to run `cargo build --target armv7-linux-androideabi` fails with

```
error[E0512]: transmute called with types of different sizes
  --> /Users/realpassy/.cargo/registry/src/github.com-1ecc6299db9ec823/jni-0.7.0/src/wrapper/objects/jvalue.rs:53:13
   |
53 |             ::std::mem::transmute::<_, usize>(val)
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: source type: jni_sys::jvalue (64 bits)
   = note: target type: usize (32 bits)
```

I'm too much of a Rust n00b to say whether or not this is an intended limitation, but it seems that v0.6.1 works fine for that target.

**Comment by jechas01** [comment]
Ah, yes, this is definitely a bug that was fixed pretty early but has apparently resurfaced. I'll put in a fix.

**Comment by jechas01** [comment]
OK, should be fixed now and pushed in 0.7.1 on crates.io!

**Comment by passy** [comment]
That was super quick. Thanks a lot! :)
